src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    newRobot[robot[key]] = key;
+  }
+
+  const oldRobotKeys = Object.values(robot).sort();
+  const newRobotKeys = Object.keys(newRobot).sort();
+
+  if (newRobotKeys.length !== oldRobotKeys.length) {
+    return null;
+  }
```

Comment:
```suggestion
  for (const key in robot) {
    if (newRobot.hasOwnProperty(robot[key])) {
      return null;
    }
    
    newRobot[robot[key]] = key;
  }
```
Try this
Also, it's better to create a variable for robot[key] as it's used twice